fastrtps_cmake_module/cmake/Modules/FindFastRTPS.cmake

```diff
@@ -36,9 +36,65 @@ set(FastRTPS_FOUND FALSE)
 find_path(FastRTPS_INCLUDE_DIR
   NAMES fastrtps/)
 
+find_package(fastcdr REQUIRED CONFIG)
+find_package(fastrtps REQUIRED CONFIG)
+
+string(REGEX MATCH "^[0-9]+\\.[0-9]+" fastcdr_MAJOR_MINOR_VERSION "${fastcdr_VERSION}")
+string(REGEX MATCH "^[0-9]+\\.[0-9]+" fastrtps_MAJOR_MINOR_VERSION "${fastrtps_VERSION}")
+
+find_library(FastCDR_LIBRARY_RELEASE
+    NAMES fastcdr-${fastcdr_MAJOR_MINOR_VERSION} fastcdr)
```

Comment:
Fixed in https://github.com/ros2/rmw_fastrtps/pull/116/commits/fd352fed1e5bf47e7cd4ddd2f2f4f6cb3e73477e